MapboxNavigation/InstructionsCardViewController.swift

```diff
@@ -177,7 +181,14 @@ open class InstructionsCardViewController: UIViewController {
     
     func snapToIndexPath(_ indexPath: IndexPath) {
         guard let itemCount = steps?.count, itemCount >= 0 && indexPath.row < itemCount else { return }
-        instructionsCardLayout.collectionView?.scrollToItem(at: indexPath, at: .left, animated: true)
+        let direction: UICollectionView.ScrollPosition = UIApplication.shared.userInterfaceLayoutDirection == .leftToRight ? .left : .right
+        if #available(iOS 14.0, *) {
+            instructionsCardLayout.collectionView?.isPagingEnabled = false
+            instructionsCardLayout.collectionView?.scrollToItem(at: indexPath, at: direction, animated: true)
+            instructionsCardLayout.collectionView?.isPagingEnabled = true
```

Comment:
This portion of code is mostly duplication. We can extract it into separate function.